bottlecap/src/config/mod.rs

```diff
@@ -260,7 +261,12 @@ pub fn get_config(config_directory: &Path) -> Result<Config, ConfigError> {
     if config.site.is_empty() {
         config.site = "datadoghq.com".to_string();
     }
-
+    if let Ok(no_proxy_string) = std::env::var("NO_PROXY") {
```

Comment:
Should we add a `todo` comment?

Reply:
For potentially parsing the entire string?